SU-HSDO/suhumsci
# SHS-5091: A11y Level A: Empty heading, on view on Bing Nursery

# READY FOR REVIEW

## Summary
Fix Empty heading, on view on Bing Nursery

## Need Review By (Date)
04/19

## Urgency
medium

## Steps to Test
1. Go to this page https://bingschool.suhumsci.loc/news-events/bing-times
2. Confirm there isn't more than one heading level on view on Bing Nursery 

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
